### PR TITLE
Update design for DesktopComparisonTable

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
- Add support for showing Peril color next to title
- Move accordion arrow to the right of title

https://github.com/user-attachments/assets/f545f9d6-4e47-4166-b0e9-17f7f57fda1d

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4677?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4677** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4677?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4673** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4673?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4671** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4671?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4668** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4668?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gustaveen and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/ui/src/theme/colors/colors.ts [hunk 1]
```diff
@@ -314,6 +314,9 @@ export const colors = {
   backgroundFrostedGlass: 'hsla(0, 0%, 98%, 0.75)',
 
   // Border
+  borderPrimary: grayTranslucent[200],
+  borderSecondary: grayTranslucent[300],
+  // Deprecated
   borderOpaque1: gray[200],
```
Comment (gustaveen) [inline packages/ui/src/theme/colors/colors.ts:320]: Will check with Nils if the opaque borders should be removed

File: apps/store/src/components/ComparisonTable/ComparisonTable.tsx [hunk 2]
```diff
@@ -56,19 +57,29 @@ export function Header({ children, active, ...props }: HeaderProps) {
...
 
-export function TitleDataCell({ className, title, description, ...props }: TitleDataCellProps) {
+export function TitleDataCell({
+  className,
+  title,
+  description,
+  perilColor,
+  ...props
+}: TitleDataCellProps) {
   const Title = (
```
Comment (alebedev) [inline apps/store/src/components/ComparisonTable/ComparisonTable.tsx:70]: nit: `title` - it's JSX element, not a component

**Review by alebedev — APPROVED** [review]
Looks good

**Review by alebedev — APPROVED** [review]

**Comment by gustaveen** [comment]
### Merge activity

* **Sep 5, 5:06 AM EDT**: @gustaveen started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4677).

**Pull request merged**